core-bundle/src/EventListener/DataContainer/PageRoutingListener.php

```diff
@@ -54,14 +54,13 @@ public function generateRouteConflicts(DataContainer $dc): string
             return '';
         }
 
-        $aliasPages = $pageAdapter->findSimilarByAlias($currentPage);
-
-        if (null === $aliasPages) {
+        if (!$aliasPages = $pageAdapter->findSimilarByAlias($currentPage)) {
             return '';
         }
 
         $conflicts = [];
-        $currentUrl = $this->buildUrl($currentPage->alias, $currentPage->urlPrefix, $currentPage->urlSuffix);
+        $currentRoute = $this->pageRegistry->getRoute($currentPage);
```

Comment:
I think `PageRegistry::getRoute()` could throw an exception here that we should handle.

Reply:
Are you sure? Your original code did not handle potential exceptions in the `getRoute()` method, either. See line 82.